react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -218,20 +234,34 @@ class ParticipantView extends Component<Props> {
                     // rendered.
                     && _toBoolean(this.props.showVideo, true)
                     && <VideoTrack
+                        onPress = { onPress }
                         videoTrack = { videoTrack }
                         waitForVideoStarted = { waitForVideoStarted }
-                        zOrder = { this.props.zOrder } /> }
+                        zOrder = { this.props.zOrder }
+                        zoomEnabled = { zoomEnabled } /> }
 
                 { renderAvatar
 
                     // The consumer of this ParticipantView is allowed to forbid
                     // showing the avatar if the private logic of this
                     // ParticipantView determines that the avatar could be
                     // rendered.
+                    //
+                    // Note: We add an onPress event here, as we moved the
+                    // onPress detection from Conference to VideoTransform (see
+                    // comment in VideoTransform#_onGesture) and we need another
+                    // component to detect touches when the video is not
+                    // rendered.
                     && _toBoolean(this.props.showAvatar, true)
-                    && <Avatar
-                        size = { this.props.avatarSize }
-                        uri = { avatar } /> }
+                    && ((onPress && <TouchableWithoutFeedback
+                        onPress = { onPress }>
```

Comment:
There's another problem with the `onPress`:
1. When the `TintView` is rendered, pressing stops working/toggling the toolbox.
2. Pressing on the "connectivity info label" doesn't toggle the toolbox.

Reply:
Not sure now if you fixed this with the recent commit or it's still an issue. I tested the connectivity issue and pressing the screen when the spinner is on seems to work fine (i.e. brings up the controls).

Which one is the connectivity info label?

Reply:
I have solved the regressions. The connectivity info label is visible in the source code.